test/acceptance/open_and_close.robot

```diff
@@ -63,3 +63,29 @@ Closing all browsers clears cache
     Run Keyword And Expect Error
     ...    No browser with index or alias 'Browser 2' found.
     ...    Switch Browser    Browser 2
+
+Click Element Without Modifier
+    Open Browser    ${ROOT}/jquery.html    ${BROWSER}
+    Set Selenium Implicit Wait    60s
```

Comment:
Why the implicit wait is increased to 60 seconds?

Reply:
I had kept it with the objective if there is a delay to load elements for the site to be loaded.
However removing it also works !!